packages/manager/modules/web-paas/src/details/user-licences/user-licences.controller.js

```diff
@@ -22,4 +31,31 @@ export default class {
   static getChangeOfferState(projectId) {
     return `web-paas.dashboard.service.change-offer({ projectId: '${projectId}'})`;
   }
+
+  trackImpression() {
+    set(
+      this.CHANGE_OFFER_IMPRESSION_TRACKING_DATA,
+      'variant',
+      `[${this.project.offerName}]`,
+    );
+    set(
+      this.CHANGE_OFFER_IMPRESSION_TRACKING_DATA,
+      'format',
+      `[${this.project.isStartOffer() ? 'develop-expand' : 'expand'}]`,
+    );
```

Comment:
Is it needed to override some values, could we extend instead 
```suggestion
    const trackingData = {
    ...CHANGE_OFFER_IMPRESSION_TRACKING_DATA,
    variant: ...
    }
```